plugins/node/opentelemetry-instrumentation-ioredis/src/instrumentation.ts

```diff
@@ -121,11 +125,8 @@ export class IORedisInstrumentation extends InstrumentationBase {
       const span = instrumentation.tracer.startSpan(cmd.name, {
         kind: SpanKind.CLIENT,
         attributes: {
-          [SemanticAttributes.DB_SYSTEM]: DbSystemValues.REDIS,
-          [SemanticAttributes.DB_STATEMENT]: dbStatementSerializer(
-            cmd.name,
-            cmd.args
-          ),
+          [SEMATTRS_DB_SYSTEM]: DbSystemValues.REDIS,
```

Comment:
This `DbSystemValues` usage should be udpated as well, I think.

Reply:
Updated